src/components/graph/menus/network-modification-node-editor.js

```diff
@@ -939,7 +985,16 @@ const NetworkModificationNodeEditor = () => {
                 >
                     <DeleteIcon />
                 </IconButton>
+                <IconButton
```

Comment:
Would it be possible to disable the button when there's no modification to restore ?

Reply:
I thought about it also but maybe it's not necessary to make a call every time to check if we disable this button + we would need to make sure the button is on the right state of disabled or not for not much. At worst the popup shows no modif to restore. But maybe it's more of a functionnal choice to make

Reply:
I suggest in this case to add a message informing thaT there is no modifications to restore into  the dialog and disable the button "restore".

Reply:
this suggestion is validated by PO